microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase.yml

```diff
@@ -0,0 +1,136 @@
+      `unknownProperty` from the host proxied object. But `proxy.getLocalProperty('unknownProperty')` gets the value of
+      the property `unknownProperty` on the proxy object.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'getLocalProperty(propertyName: string): any;'
+      parameters:
+        - id: propertyName
+          description: Name of the property to get the value of.
+          type: string
+      return:
+        type: any
+        description: A promise representing the value of the property.
+  - name: 'setHostProperty(propertyName, propertyValue)'
+    uid: 'WebView2Script!HostObjectAsyncProxyBase#setHostProperty:member(1)'
+    package: WebView2Script!
+    fullName: 'setHostProperty(propertyName, propertyValue)'
+    summary: >-
+      Perform a property `set` on the host object. Use this method to explicitly force a property `set` to occur remotely if
+      a conflicting local method or property exists.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'setHostProperty(propertyName: string, propertyValue: any): Promise<any>;'
+      parameters:
+        - id: propertyName
+          description: Name of the property of which to set the value.
+          type: string
+        - id: propertyValue
+          description: Value to set the property.
+          type: any
+      return:
+        type: Promise&lt;any&gt;
+        description: >-
+          A promise representing the converted value of the property of the host object's property. This promise only
+          resolves after the property value has been changed.
+  - name: 'setLocalProperty(propertyName, propertyValue)'
+    uid: 'WebView2Script!HostObjectAsyncProxyBase#setLocalProperty:member(1)'
+    package: WebView2Script!
+    fullName: 'setLocalProperty(propertyName, propertyValue)'
+    summary: >-
+      Perform a property `set` locally on the proxy object. Use the methods to force setting a property on the host object
+      proxy rather than on the host object it represents. For instance, `proxy.unknownProperty = 2` sets the property
+      named `unknownProperty` on the host proxied object. But `proxy.setLocalProperty('unknownProperty', 2)` sets the
+      value of the property `unknownProperty` on the proxy object.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'setLocalProperty(propertyName: string, propertyValue: any): any;'
+      parameters:
+        - id: propertyName
+          description: Name of the property to get the value of.
+          type: string
+        - id: propertyValue
+          description: Value to set the property to.
+          type: any
+      return:
+        type: any
+        description: >-
+          A promise representing the value of the property after it is set. This promise only resolves after the
+          property changes value.
+extends: CallableFunction

```

Comment:
The above is displayed on the rendered webpage after the summary and isn't linked. I don't see a class in the ToC called CallableFunction. 

Is this a bug?

Reply:
It's expected that it's not linked. The declaration of HostObjectAsyncProxyBase in our d.ts is like below:

```
interface HostObjectAsyncProxyBase extends CallableFunction, Promise<any> {}
```

Reply:
Right. CallableFunction and Promise are types from JavaScript and for the d.ts they come from a standard TypeScript d.ts file that defines JavaScript types. These sort of types shouldn't exist in our docs. Not sure if there is anything else we do for these beyond that.

Reply:
Is this thread resolved?